### ISSUE TITLE
Add interactive option

### ISSUE THREAD

**Issue by darsain** [darsain/tooltips#6]
`interactive` option should show the tooltip when mouse enters the tooltip element (and hide when leaves).

The purpose would be (along with delay option #3) creating interactive tooltips.

It should also set the Tooltip's `interactive` option (https://github.com/darsain/tooltip/issues/10) to true.

**Comment by ghost** [comment]
Almost hitting on the 5 month mark @darsain, please tell me you still have the motivation for this.  Simply, a `data-interact` attribute could be crafted, and it would act like a menu as if the users mouse leaves, it closes.

Since HTML works inside the attributes, a user could make/generate w/e menu they'd like. I think it would be an amazing addon / idea.

**Comment by darsain** [comment]
Yeah, I just first want to finish all the stuff in queue for underlying Tooltip library before I move to this one. And when I move to this one I want to refactor it, as I don't like how it's written right now.

This is a lot of work for which I haven't had time yet (paying the bills has priority). So unless I need it in one of my commercial projects, it'll have to wait until I have a break.

Also, if virtually everyone wouldn't ignore "support the development" buttons, it would be easy to justify spending a day of work on these projects here and there :) I wish there was some business model for OSS projects that would at least cover the work done. I'm definitely not looking to get rich here... but lets not make this issue about that.

**Comment by ghost** [comment]
Yeah, I don't got much at the moment but I did send 16 USD over, not sure how much that is in euro's but it's better than nothing I guess. I am using your tooltip addon for my web based game: http://i.gyazo.com/ab4b5103a5e0225689d4c0ab933cab84.png (It's still in development, using nodejs/websockets :D)

 So I do appreciate it a lot as it's actually the PERFECT tooltip / plugin for a game. Especially when monsters will be dropping items and the user needs to pick them up, tips.add is all I need, and it's dom manipulation is just incredible and your API is so easy. I'll hopefully send over more once the game gets up and running. I know it's not much now, but I feel you. Plus, honestly  ( in my opinion ), not a lot of people give a fuck about the authors and just take shit and run.  That's how I felt and that's why I never donated anything until I saw your post, I feel bad :P  

Anyways, im not expecting you do anything, my idea was just a suggestion, your tooltips program is enough it self, and I really love it :)

Edit: (http://i.gyazo.com/fb6fcdde41aa80b3b2922d7958d47b7e.png) It looks like it converted it self. 15 Euro's it is.  I guess 2 cups of coffee.. haha not much, but just wait I will send more later.

**Comment by ghost** [comment]
Also if anyone else is viewing this thread and are using the tooltip?s script, please do donate and show your support.  I figured I'd just use it and run with it and never show any support, which you have the right to, but ethically atleast do something and give something back.  There is really no other tooltip script that doesn't RELY on JQUERY like this out in the web market today that I know of anyways. (That is free!) Don't take it for granted.

**Comment by darsain** [comment]
Donation is appreciated, but that is not what I was trying to instigate :) I was just complaining about lack of OSS funding in general. Anyway, lets not derail this issue further.
